### DISCUSSION TITLE
rustdesk on ubuntu 22.04 mysteriously stops shortly after startup

### DISCUSSION THREAD

**Discussion by pdxrlk** [rustdesk/rustdesk#6212] [Q&A]
rustdesk was working fine on one of my ubuntu machines until some time after a recent update/upgrade of various components (via apt).  When I run `systemctl status rustdesk` I get messages saying the service is disabled, and the log shows it stops just one second after it started.  Re-enabling and restarting doesn't fix it (`systemctl enable rustdesk; systemctl start rustdesk`).  Any ideas what's going on, or how to debug it further?

```
○ rustdesk.service - RustDesk
     Loaded: loaded (/lib/systemd/system/rustdesk.service; disabled; vendor preset: enabled)
     Active: inactive (dead)

Oct 28 09:44:25 benevolent2 systemd[1]: rustdesk.service: Unit process 14035 (systemctl) remains running after unit stopped.
Oct 28 09:44:25 benevolent2 systemd[1]: Stopped RustDesk.
Oct 28 09:44:57 benevolent2 systemd[1]: Started RustDesk.
Oct 28 09:44:58 benevolent2 systemd[1]: Stopping RustDesk...
Oct 28 09:44:58 benevolent2 systemd[1]: rustdesk.service: Killing process 14175 (sh) with signal SIGKILL.
Oct 28 09:44:58 benevolent2 systemd[1]: rustdesk.service: Killing process 14196 (systemctl) with signal SIGKILL.
Oct 28 09:44:58 benevolent2 systemd[1]: rustdesk.service: Deactivated successfully.
Oct 28 09:44:58 benevolent2 systemd[1]: rustdesk.service: Unit process 14175 (sh) remains running after unit stopped.
Oct 28 09:44:58 benevolent2 systemd[1]: rustdesk.service: Unit process 14196 (systemctl) remains running after unit stopped.
Oct 28 09:44:58 benevolent2 systemd[1]: Stopped RustDesk.

```

**Comment by joaocarloscabreu** [comment]
I am having exactly the same problem in ubuntu 22.04

**Comment by Moonbase59** [comment]
And here the same on EndeavourOS (Cinnamon DE, X11). Interestingly, on two almost identically set up laptops, the service only gets killed upon rustdesk start on the Thinkpad T14 Gen 1, _not_ on the Toshiba Satellite L70-B.
